kp_service_test/src/test/resources/templates/content/v3/upload/testUploadResourceEcmlWithTwinAnimationAudioSpritesImgSprites/validate.json

```diff
@@ -0,0 +1,58 @@
+{
+  "id": "ekstep.content.find",
+  "ver": "3.0",
+  "ts": "@matchesDatePattern('yyyy-MM-dd')@",
+  "params": {
+    "resmsgid": "@ignore@",
+    "msgid": null,
+    "err": null,
+    "status": "successful",
+    "errmsg": null
+  },
+  "responseCode": "OK",
+  "result": {
+    "content": {
+      "ownershipType": [
+        "createdBy"
+      ],
+      "code": "kp.ft.resource.ecml",
+      "channel": "channel-01",
+      "description": "KP Integration Test Content",
+      "language": [
+        "English"
+      ],
+      "mimeType": "application/vnd.ekstep.ecml-archive",
+      "idealScreenSize": "normal",
+      "createdOn": "@matchesDatePattern('yyyy-MM-dd')@",
+      "appId": "app-01",
+      "contentDisposition": "inline",
+      "artifactUrl": "@ignore@",
+      "contentEncoding": "gzip",
+      "lastUpdatedOn": "@matchesDatePattern('yyyy-MM-dd')@",
+      "dialcodeRequired": "No",
+      "contentType": "Resource",
+      "identifier": "@ignore@",
+      "audience": [
+        "Learner"
+      ],
+      "lastStatusChangedOn": "@matchesDatePattern('yyyy-MM-dd')@",
+      "os": [
+        "All"
+      ],
+      "visibility": "Default",
+      "consumerId": "@ignore@",
+      "mediaType": "content",
+      "osId": "org.ekstep.quiz.app",
+      "languageCode": "en",
+      "version": 2,
+      "versionKey": "@ignore@",
```

Comment:
Please use isNumber function for all numeric fields.

Reply:
Fixed